profiles/tests/test_viewed_company.py

```diff
@@ -0,0 +1,77 @@
+from rest_framework.test import APITestCase, APIClient
+from authentication.models import CustomUser
+from profiles.models import Profile
+from utils.dump_response import dump # noqa
+
+
+class TestViewedCompanyAPI(APITestCase):
+    def setUp(self) -> None:
+        self.test_person = CustomUser.objects.create_user(
+            person_email="test1@test.com",
+            password="Testing01",
+            person_name="test1",
+            person_surname="test",
+            is_active=True
+        )
+        test_person2 = CustomUser.objects.create_user(
+            person_email="test2@test.com",
+            password="Testing01",
+            person_name="test2",
+            person_surname="test",
+            is_active=True
+        )
+        test_person3 = CustomUser.objects.create_user(
+            person_email="test3@test.com",
+            password="Testing01",
+            person_name="test3",
+            person_surname="test",
+            is_active=True
+        )
+        self.test_profile = Profile.objects.create(
+            person=self.test_person,
+            comp_is_startup=True,
+            comp_registered=False
+        )
+        self.test_profile2 = Profile.objects.create(
+            person=test_person2,
+            comp_registered=True,
+            comp_is_startup=False
+        )
+        self.test_profile3 = Profile.objects.create(
+            person=test_person3,
+            comp_registered=True,
+            comp_is_startup=False
+        )
+
+        self.client = APIClient()
```

Comment:
Please DON'T do like this.

If the class is derived from `APITestCase`, you already have `self.client` properly set.

Reply:
Removed.